src/app/client/src/app/modules/learn/components/course-consumption/course-consumption-header/course-consumption-header.component.html

```diff
@@ -54,6 +54,17 @@ <h5 class="textbook__title sb-color-primary font-weight-bold mb-0">{{courseHiera
                 </svg></i>
                 {{resourceService?.frmelmnts?.lbl?.copy}}
               </button>
+
+               <!-- Discussion Forum -->
+               <button class="sb-btn sb-btn-normal sb-btn-link sb-btn-link-primary sb-left-icon-btn ml-8 sb-btn-icon-fix" 	
+               appTelemetryInteract [telemetryInteractObject]="courseInteractObject"	
+               [telemetryInteractEdata]="{id:'discussion' , type: 'click' , pageid:'course-consumption'}" 	
+               [telemetryInteractCdata]="[{id: courseHierarchy?.identifier, type: 'Course'}]"	
+               (click)="openDiscussionForum()" *ngIf="forumId && (enrolledCourse || (courseConsumptionService?.userCreatedAnyBatch | async))"	
+               title="Forum">	
+                 <img src="assets/images/forum.svg" width="16px" class="mr-8">	
+                 Forum	
```

Comment:
please remove the hard coded value and add a label

Reply:
This is how it went to prod last time because of shottime. Agreed with PO to go without translations. 
But I have added for now.